vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/DeviceSteps.java

```diff
@@ -76,21 +80,68 @@ public void uploadFileToDevice(String filePath)
      * See <a href="https://github.com/appium/appium-xcuitest-driver#mobile-pressbutton">iOS keys</a> and
      * <a href="https://appium.github.io/java-client/io/appium/java_client/android/nativekey/AndroidKey.html">
      * Android keys</a> for available values
+     * <br>
+     * Example:
+     * <br>
+     * <code>
+     * When I press $key key
+     * </code>
      *
-     * @param key key to press
+     * @param key the key to press
      */
     @When("I press $key key")
     public void pressKey(String key)
+    {
+        pressKeys(List.of(key));
+    }
+
+    /**
+     * Presses the keys
+     * <br>
+     * See <a href="https://github.com/appium/appium-xcuitest-driver#mobile-pressbutton">iOS keys</a> and
+     * <a href="https://appium.github.io/java-client/io/appium/java_client/android/nativekey/AndroidKey.html">
+     * Android keys</a> for available values
+     * <br>
+     * Example:
+     * <br>
+     * <code>
+     * When I press keys:
+     * <br>
+     * |key |
+     * <br>
+     * |Home|
+     * </code>
+     *
+     * @param keys the keys to press
+     */
+    @When("I press keys:$keys")
+    public void pressKeys(ExamplesTable keys)
+    {
+        pressKeys(MapUtils.convertExamplesTableToMap(keys).get("key"));
```

Comment:
does it make sense to introduce new method in `ExamplesTable`: `getColumn(String key)` ?

Reply:
Added https://github.com/jbehave/jbehave-core/commit/390e2a2bd3abcbe21dc082bc1128af3711b17fd2